### PR TITLE
Thaikum navbar

### PR DESCRIPTION
**Pull request by thaikum**
Check my code

### CODE REVIEW

**Comment by kiambati-maxwel** [comment]
its awsome we shoul add one more button

**Comment by thaikum** [comment]
Aw i get it i think alink is better than a button

**Comment by kiambati-maxwel** [comment]
okay let me add it

**Comment by kiambati-maxwel** [comment]
check it out ...

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (kiambati-maxwel) [inline index.html:4]: set it to ASCII

**Review by kiambati-maxwel — COMMENTED** [review]
niice

File: index.html (hunk 1 above)
Comment (thaikum) [reply index.html:4]: Changed

**Pull request merged**